contracts/SimpleGAMultiSig.aes

```diff
@@ -123,11 +124,16 @@ contract SimpleGAMultiSig =
     require_valid_signer_and_tx_hash(Call.caller, tx_hash, tx)
     require(!Set.member(Call.caller, tx.confirmed_by), "ERROR_ALREADY_CONFIRMED")
     require(!expired(tx), "ERROR_TX_EXPIRED")
-    let updated_tx = tx{ confirmed_by = Set.insert(Call.caller, tx.confirmed_by) }
-    put(state{ current_tx = Some(updated_tx) })
-    Chain.event(TxConfirmed(tx.tx_hash, Call.caller))
-    if(state.confirmations_required == Set.size(updated_tx.confirmed_by))
-      Chain.event(TxConsensusReached(tx.tx_hash))
+    //
+    switch(Set.member(Call.caller, tx.refused_by))
+      true =>
+        let updated_tx = tx{ refused_by = Set.delete(Call.caller, tx.refused_by),
```

Comment:
why is this needed? I guess it should be ok to have my address as confirmed and refused in order for others to make the decision, even if not needed it doesn't hurt and will be cheaper in gas

Reply:
we need it to determine what to display in the UI (accepted/rejected) cause now it's not possible to know that while the same address is in both `approvedBy` and `refusedBy`

Reply:
can't both be shown?

Reply:
I think we should build the UI to serve the contract, not the contract to serve the UI

Reply:
I think it is more a logical thing. users would be very confused to see a signer in both lists

Reply:
as discussed with @ifaouibadi it will be important to revoke accidental approval, so this change is different logic but probably better.